sv.c

```diff
@@ -4972,8 +4972,9 @@ Perl_sv_setsv_cow(pTHX_ SV *dsv, SV *ssv)
 /*
 =for apidoc sv_setpv_bufsize
 
-Sets the SV to be a string of cur bytes length, with at least
-len bytes available. Ensures that there is a null byte at SvEND.
+Sets the SV to be a string of C<cur> bytes length, with at least
+C<len> bytes available.   If <cur> is not zero, the string contents are
+unspecified.  Ensures that there is a null byte at C<SvEND>.
```

Comment:
the second `<cur>` is missing the `C`

Maybe:
```
If C<cur> is not zero the string contents are not initialized.
```
Or (this is my preferred choice):
```
The string contents will not be initialized.
```
Since `cur == 0` is just a degenerate case and "will not be" makes it less passive.

If we want to make fewer promises, so we could change it to initialize in some cases (`-DDEBUGGING` builds might initialize it to a filler byte.)
```
The string contents currently will not be initialized.  This may change.
```

Reply:
I changed it to be more explicit of what the caller needs to do.  I think that obviates the need to say we could change it, because any change we make would not be harder for the caller than what is now called for.